pdr_backend/lake/test/test_fetch_ohlcv.py

```diff
@@ -87,6 +92,17 @@ def test_safe_fetch_ohlcv(exch):
     v = safe_fetch_ohlcv_ccxt("bad exch", symbol, timeframe, since, limit)
     assert v is None
 
+    # happy path dydx
+    fifteen_min_ago = datetime.now(timezone.utc) - timedelta(minutes=15)
+    symbol, timeframe, since, limit = (
+        "BTC-USD",
+        "5MINS",
+        UnixTimeMs.from_dt(fifteen_min_ago),
+        100,
+    )
+    result = safe_fetch_ohlcv_dydx("dydx", symbol, timeframe, since, limit)
+    assert_safe_fetch_ohlcv_dydx_ok(result)
```

Comment:
Why is this a separate function, vs just having the three lines here? Are you going to reuse it for more tests? 
- If yes, ok. 
- If no, move the three lines of the function into here.